commit b7883ae1398499daaa926ae6621f088f0f531ed8
Author: Mans Rullgard <mans@mansr.com>
Date:   Wed Apr 24 16:56:42 2019 +0100

    fft4g: bail if size too large (CVE-2019-8356)
    
    Prevent overflowing of fixed-size buffers in bitrv2() and bitrv2conj()
    if the transform size is too large.

diff --git a/src/fft4g.h b/src/fft4g.h
index 2b8051ca..95ee3413 100644
--- a/src/fft4g.h
+++ b/src/fft4g.h
@@ -13,6 +13,8 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
 
+#define FFT4G_MAX_SIZE 262144
+
 void lsx_cdft(int, int, double *, int *, double *);
 void lsx_rdft(int, int, double *, int *, double *);
 void lsx_ddct(int, int, double *, int *, double *);
